prow/config.yaml

```diff
@@ -212,8 +212,18 @@ tide:
     - do-not-merge/work-in-progress
     - needs-ok-to-test
     - needs-rebase
+  - repos:
+    - istio/operator
+    labels:
+    cla: "yes"
```

Comment:
This still doesn't match how it's done above. Just copy how we do it there (this also won't work)